lighthouse-core/audits/dobetterweb/uses-request-compression.js

```diff
@@ -48,29 +48,36 @@ class UsesRequestCompression extends Audit {
   static audit(artifacts) {
     const networkRecords = artifacts.networkRecords[Audit.DEFAULT_PASS];
 
-    // Filter requests that are on the same host as the page and not over h2.
+    // Filter requests that are text based and have gzip/br encoding.
     const resources = networkRecords.filter(record => {
       return record._resourceType && record._resourceType._isTextType &&
-        !record._responseHeaders.find(header =>
-          header.name.toLowerCase() === 'content-encoding' &&
-          (header.value === 'gzip' || header.value === 'br')
-        );
+          record._resourceSize > 1000 &&
```

Comment:
I interpreted this as a safeguard against small files that might increase in file size due to gzipping since we aren't actually gzipping them to see if they'd be smaller, so I'm still in favor of having some sort of threshold on the size.